lib/cartopy/io/img_tiles.py

```diff
@@ -44,17 +44,24 @@ class GoogleTiles(object):
     A "tile" in this class refers to the coordinates (x, y, z).
 
     """
-    def __init__(self, desired_tile_form='RGB', style="street"):
+    def __init__(self, desired_tile_form='RGB', style="street",
+                url=('https://mts0.google.com/vt/lyrs={style}' \
+                     '@177000000&hl=en&src=api&x={x}&y={y}&z={z}&s=G')):
         """
         :param desired_tile_form:
         :param style: The style for the Google Maps tiles. One of 'street',
             'satellite', 'terrain', and 'only_streets'.
             Defaults to 'street'.
+        :param url: str url pointing to a tile source such as in default
+                    {x}, {y}, and {z} must be included in url. Such as:
+                    ('https://server.arcgisonline.com/ArcGIS/rest/services/' \
+                     'World_Shaded_Relief/MapServer/tile/{z}/{y}/{x}.jpg')
         """
         # Only streets are partly transparent tiles that can be overlayed over
         # the satellite map to create the known hybrid style from google.
         styles = ["street", "satellite", "terrain", "only_streets"]
         style = style.lower()
+        self.url = url.lower()
```

Comment:
I had that in there to force the x, y, z to lowercase, so that the user could use upper or lower case. But I just have the format catching upper and lower now.